lib/traveler.rb

```diff
@@ -0,0 +1,10 @@
+module RideShare
+  class Traveler
+    attr_reader :trips
```

Comment:
I'm not sure what this class is for - it doesn't look like you use it anywhere.

It looks like maybe you were planning on making it a common superclass for `Driver` and `Passenger`, so that the `finished_trips` method doesn't need to be repeated. If so, this is an interesting idea, and I applaud the effort to keep your code DRY.